AppController/djinn.rb

```diff
@@ -3551,10 +3551,19 @@ def prime_database()
     retries = 10
     loop {
       return if system("#{prime_script} --replication #{replication}")
-      Djinn.log_warn("Failed to prime database. #{retries} retries left.")
-      Kernel.sleep(SMALL_WAIT)
       retries -= 1
+      Djinn.log_warn("Failed to prime database. #{retries} retries left.")
+
+      # Run a repair and try to prime one more time.
+      if retries == 1
+        @state = 'Running a Cassandra repair.'
+        Djinn.log_warn(@state)
+        system("#{NODETOOL} repair")
```

Comment:
I can't find anything in the docs about this, and I'm not sure even if this will prevent the issue from happening. There's just a couple SO and mailing list posts that suggest it might fix the issue. I'll add another comment explaining what the issue is, though.